src/main/java/org/folio/innreach/domain/dto/folio/inventory/InventoryItemStatus.java

```diff
@@ -22,14 +23,17 @@
   AWAITING_DELIVERY("Awaiting delivery"),
   MISSING("Missing"),
   ON_ORDER("On order"),
+  ORDER_CLOSED("Order closed"),
   IN_PROCESS("In process"),
   CLAIMED_RETURNED("Claimed returned"),
   LOST_AND_PAID("Lost and paid"),
   INTELLECTUAL_ITEM("Intellectual item"),
   IN_PROCESS_NON_REQUESTABLE("In process (non-requestable)"),
   UNAVAILABLE("Unavailable"),
   UNKNOWN("Unknown"),
-  RESTRICTED("Restricted");
+  RESTRICTED("Restricted"),
+  RETRIEVING_FROM_ASR("Retrieving from ASR"),
+  MISSING_FROM_ASR("Missing from ASR");
```

Comment:
Are these statuses existing in inventory API? I can't find them on ramls

Reply:
Those statuses will be implemented later, so I think we can add them now.